src/tunacode/tools/utils/ripgrep.py

```
@@ -22,34 +23,42 @@
+
     system = platform.system().lower()
     machine = platform.machine().lower()
 
     if system == "linux":
         if machine in ["x86_64", "amd64"]:
+            ripgrep_cache.set_platform_identifier("x64-linux", system)
             return "x64-linux", system
         elif machine in ["aarch64", "arm64"]:
+            ripgrep_cache.set_platform_identifier("arm64-linux", system)
             return "arm64-linux", system
     elif system == "darwin":  # noqa: SIM102
```

Ruff (0.14.14)

[warning] Unused `noqa` directive (non-enabled: `SIM102`)

Remove unused `noqa` directive

(RUF100)